package.json

```diff
@@ -10,14 +10,23 @@
     "WordPress",
     "editor"
   ],
+  "files": [
+    "index.php",
+    "post-content.js",
+    "editor/build",
+    "i18n/build",
+    "element/build",
+    "blocks/build"
+  ],
   "scripts": {
     "test-unit": "cross-env NODE_ENV=test webpack && mocha build --require bootstrap-test.js",
     "build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
     "gettext-strings": "cross-env BABEL_ENV=gettext webpack",
     "lint": "eslint .",
     "dev": "cross-env BABEL_ENV=default webpack --watch",
     "test": "npm run lint && npm run test-unit",
-    "ci": "concurrently \"npm run build\" \"npm test\""
+    "ci": "concurrently \"npm run build\" \"npm test\"",
+    "build-plugin": "rm -f gutenberg.zip && npm install && npm run build && cat \"$(npm pack)\" | tar xzf - && cd package && zip -r ../gutenberg.zip * && cd - && rm -rf package gutenberg*.tgz"
```

Comment:
Good idea, I wasn't aware that npm run the `pre` and `post` automatically. Such a shame :). Updating